src/components/trade/LastTrades/LastTrades.tsx

```diff
@@ -0,0 +1,70 @@
+import React from 'react'
+import BigNumber from 'bignumber.js'
+import { TokenDetails } from 'types'
+import styled from 'styled-components'
+import { formatPrice } from '@gnosis.pm/dex-js'
+import Spinner from 'components/common/Spinner'
+import ErrorMsg from 'components/ErrorMsg'
+import { formatDateLocaleShortTime } from 'utils'
+
+const Wrapper = styled.div`
+  li {
+    padding: 0.5rem;
+  }
+`
+
+export interface Props {
+  quoteToken: TokenDetails
+  trades: LastTradesItem[]
+  error?: Error
+  loading: boolean
+}
+
+export interface LastTradesItem {
+  id: string
+  size: BigNumber
+  price: BigNumber
+  time: Date
+}
+
+export const LastTrades: React.FC<Props> = (props) => {
+  const { loading, error, trades } = props
+  const lastTrades = trades.map((trade) => {
+    const { id, size, price, time } = trade
+    return (
+      <li key={id}>
+        {formatPrice(size)} - {formatPrice(price)} - {formatDateLocaleShortTime(time.getTime())}
+      </li>
+    )
+  })
```

Comment:
memoize?

Reply:
Happy to do it, but for this early state I would consider it also premature optimization